model/model/v1alpha/model.proto

```diff
@@ -1737,3 +1737,92 @@ message UndeployOrganizationModelAdminRequest {
 
 // UndeployOrganizationModelAdminResponse represents a response for a undeployed model
 message UndeployOrganizationModelAdminResponse {}
+
+// ModelVersion contains information about run log of a model.
+message ModelRunLogging {
+  // RunStatus defines the status of a model run
+  enum RunStatus {
+    // Unspecified
+    RUN_STATUS_UNSPECIFIED = 0;
+    // model run in progress
+    RUN_STATUS_PROCESSING = 1;
+    // model run succeeded
+    RUN_STATUS_COMPLETED = 2;
+    // model run failed
+    RUN_STATUS_FAILED = 3;
+    // model run is waiting to be executed
+    RUN_STATUS_QUEUED = 4;
+  }
+
+  // Namespace ID
+  string namespace_id = 1 [(google.api.field_behavior) = REQUIRED];
+  // Model UUID.
+  string model_uid = 2 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger UUID.
+  string trigger_uid = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model run logging status
+  RunStatus status = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model visibility.
+  Model.Visibility visibility = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger source
+  string source = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger start time
+  google.protobuf.Timestamp start_time = 7 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Trigger total duration in milliseconds
+  optional int64 total_duration = 8 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger end time
+  optional google.protobuf.Timestamp end_time = 9 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Trigger requesters
+  string runner = 10 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Credit consumptions
+  optional float credits = 11 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Error msg occurred during model run
+  optional string errors = 12 [
+    (google.api.field_behavior) = OUTPUT_ONLY,
+    (google.api.field_behavior) = OPTIONAL
+  ];
+  // Model run logging creation time.
+  google.protobuf.Timestamp create_time = 13 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // Model run logging update time.
+  google.protobuf.Timestamp update_time = 14 [(google.api.field_behavior) = OUTPUT_ONLY];
+}
```

Comment:
I'm missing the tag/version field here, in the UX it's present.

Reply:
updated